src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsalertsapi/resource/AlertsController.kt

```diff
@@ -195,6 +193,10 @@ class AlertsController(
         description = "Forbidden, requires an appropriate role",
         content = [Content(schema = Schema(implementation = ErrorResponse::class))],
       ),
+      ApiResponse(
+        responseCode = "404",
+        description = "Alert was not found or already deleted",
+      ),
```

Comment:
Think you need `content = [Content(schema = Schema(implementation = ErrorResponse::class))],` so that the docs will show the correct response type

Reply:
Done.